ui/src/groups/ChannelsList/ChannelManagerHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import { useIsMobile } from '@/logic/useMedia';
+import { useAmAdmin, useRouteGroup } from '@/state/groups';
+import cn from 'classnames';
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import ChannelsListSearch from './ChannelsListSearch';
+
+interface ChannelManagerHeaderProps {
+  addSection: () => void;
+}
+
+export default function ChannelManagerHeader({
+  addSection,
+}: ChannelManagerHeaderProps) {
+  const location = useLocation();
+  const flag = useRouteGroup();
+  const isAdmin = useAmAdmin(flag);
+  const isMobile = useIsMobile();
+
+  return (
+    <div
```

Comment:
we should prefer using tailwind for responsive stuff where we can

Reply:
Agreed. Worth going through these changes to replace all of the isMobile references?

Reply:
Fixed.